src/components/name/more-detail.js

```diff
@@ -176,7 +175,7 @@ const AddMoreDetail = (props) => {
                 >
                   Update Alias
                 </button>
-               {/* <ListAlias/> */}
+                {/* <FormModal /> */}
```

Comment:
Am not quite sure how this is working. Also it looks like there could be missing files that you haven't added? I assume the FormModal is component and I don't see it added.